docs/integrations/index.md

```diff
@@ -110,6 +110,8 @@ orchestrated by Flyte itself, within its provisioned Kubernetes clusters.
   - Execute tasks using the MemVerge Memory Machine Cloud agent.
 * - {doc}`OpenAI Batch </auto_examples/openai_batch_agent/index>`
   - Submit requests for asynchronous batch processing on OpenAI.
+* - {doc}`PERIAN Agent </auto_examples/perian_agent/index>`
```

Comment:
Heads up that you will also need to add an entry for this agent to the `toctree` at the bottom of this file.